productcat/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,11 @@
 <resources>
     <string name="app_name">ViSearch</string>
+    <string name="policy_title">Welcome to Visual Shopping</string>
+    <string name="policy_content">To use Visual Shopping, we will continually analyze what you see to show your results. When you stop using the feature, we will stop analyzing</string>
+    <string name="agree_terms">I agree to <a href="https://www.visenze.com/company/privacy-policy">Visenze\'s Privacy Policy and Terms of Use</a></string>
```

Comment:
let's update this with See Rong's version. Our terms does not seem to be online yet.

Reply:
changed to seerong's url already. currently the url is not accessible yet.